plugins/action/ci_script.py

```diff
@@ -109,7 +115,7 @@
 %(opts)s
 exec > >(tee -i %(logpath)s) 2>&1
 %(pushcmd)s
-%(content)s
+%(params)s%(content)s
```

Comment:
as discussed on Friday's reviews call, I'll update this to use exports instead

also renaming params to extra_args to prevent any confusion with the ansible.community.make module's params